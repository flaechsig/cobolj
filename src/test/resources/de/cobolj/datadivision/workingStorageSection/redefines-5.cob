IDENTIFICATION DIVISION.
PROGRAM-ID. PERFTIMI.
ENVIRONMENT DIVISION.
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 Birth-Date-YYYYMMDD    pic 9(8) value 19991212.
  01 Birth-Data-Separate redefines Birth-Date-YYYYMMDD.
     05 Birth-Date-YYYY     pic 9(4).
     05 Birth-Date-MM       pic 99.
     05 Birth-Date-DD       pic 99.
  
PROCEDURE DIVISION.
	DISPLAY Birth-Date-YYYYMMDD.
	DISPLAY Birth-Date-YYYY.
	DISPLAY Birth-Date-MM.
	DISPLAY Birth-Date-DD.
	
	STOP RUN.