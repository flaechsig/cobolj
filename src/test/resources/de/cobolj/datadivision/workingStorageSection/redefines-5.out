19991212
1999
12
12
